src/modules/Users.pm

```diff
@@ -1532,9 +1397,6 @@ sub ReadAvailablePlugins {
 		}
 	    }
 	}
-	UsersLDAP->SetUserInternal (\@user_internals);
-    }
-    my @group_internals	= @{UsersLDAP->GetGroupInternal ()};
```

Comment:
Here the declaration is removed, but not the use of @group_internals in the following lines. Not sure what to do, honestly, if remove those lines or re-declare @group_internals to be an empty hash (!!??)

@jreidinger, @joseivanlopez could you help here?

Reply:
well, but on other hand as it is local variable and also set is removed, then I think that maybe we can remove complete code also with traversing params.